modules/home/ui/navbar.tsx

```
@@ -65,3 +67,7 @@ function NavItem({ href, text }: NavItemProps) {
     </li>
   );
 }
+
+function MobileNav(){
```

ESLint

[error] 'MobileNav' is defined but never used.

(@typescript-eslint/no-unused-vars)